Pilot/pilotTools.py

```diff
@@ -1114,9 +1114,8 @@ def __getSearchPaths(self):
         vo = self.__getVO()
         paths = [
             "/Defaults/Pilot",
-            "/%s/Pilot" % self.setup,
+            "/%s/Pilot" % vo,
             "/%s/Defaults/Pilot" % vo,
-            "/%s/%s/Pilot" % (vo, self.setup),
         ]
```

Comment:
We need to be compatible also with version of the CS that still have self.setup in it.

Reply:
OK, last commit removed.